config/nativephp.php

```diff
@@ -58,12 +58,12 @@
         'providers' => [
             's3' => [
                 'driver' => 's3',
-                'key' => env('AWS_ACCESS_KEY_ID'),
-                'secret' => env('AWS_SECRET_ACCESS_KEY'),
+                'profile' => env('AWS_PROFILE'),
```

Comment:
Why are you doing this instead of the key/secret?

Reply:
Because I didn't know if the key / secret were compiled in the app so I opted for the profile